platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -115,8 +115,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
I think we should postpone upgrading to Java 8 until the new PR with followup changes to https://github.com/mapbox/mapbox-gl-native/pull/10402/